contribs/gmf/src/directives/search.js

```diff
@@ -959,7 +964,11 @@ gmf.SearchController.prototype.fulltextsearch_ = function(query, resultIndex) {
         const format = new ol.format.GeoJSON();
         const feature = format.readFeature(data.features[resultIndex - 1]);
         this.featureOverlay_.addFeature(feature);
-        this.map_.getView().fit(feature.getGeometry().getExtent());
+        const fitOptions = /** @type {olx.view.FitOptions} */ ({});
+        if (opt_zoom) {
```

Comment:
Not sure that the zoom can be `0` but in the case the test will be false.
Should be changed to `if (opt_zoom !== undefined) {`